File: Java/HotelPrices/HotelPrices.java
import java.util.Scanner;

class HotelPrices {

    public static void main(String[] args)  {
        Scanner sc = new Scanner(System.in);
        int testCases  = sc.nextInt();
        long totalCost = 0; // Cost of all the rooms in the hotel

        while (testCases-- !=0) {
            String roomType = sc.next();
            int bedrooms = sc.nextInt();
            int bathrooms = sc.nextInt();
            long roomCost = 5 * bedrooms + 2 * bathrooms;
            if (roomType.equalsIgnoreCase("luxury")) { // Add additional charges for Luxury Rooms
                roomCost += 200;
            }
            totalCost += roomCost;
        }

        System.out.println(totalCost);
    }
}

File: C - C++/Hotel Prices/HotelPrices.cpp
#include<bits/stdc++.h>
using namespace std;
int main() {
	int t;
	cin >> t;
	long long total_cost = 0;

	while (t--) {
		string room_type;
		int bedrooms, bathrooms;
		cin >> room_type >> bedrooms >> bathrooms;
		long long cost = (5 * bedrooms) + (2 * bathrooms);

		if (room_type == "luxury") {
			cost += 200;
		}
		total_cost += cost;
	}

	cout << total_cost << "\n";
	return 0;
}
